src/EditorFeatures/TestUtilities/MinimalTestExportProvider.cs

```diff
@@ -124,7 +125,7 @@ public static PartDiscovery CreatePartDiscovery(Resolver resolver)
 
         public static ExportProvider CreateExportProvider(ComposableCatalog catalog)
         {
-            var configuration = CompositionConfiguration.Create(catalog.WithDesktopSupport().WithCompositionService());
```

Comment:
WithDesktopSupport() is deprecated, see https://github.com/Microsoft/vs-mef/blob/v15.5/src/Microsoft.VisualStudio.Composition/Configuration/NetFxAdapters.cs#L59